docs/docs/commands/slashingImport copy.mdx

```diff
@@ -0,0 +1,111 @@
+---
+sidebar_position: 4
+id: slashing-export
+---
+
+# slashing-export
+
+Running `sedge slashing-export` will export slashing protection data from the current validator client. The slashing protection data is a JSON file that meets with the [EIP-3076](https://eips.ethereum.org/EIPS/eip-3076) specification.
+
+## Help
+
+```shell
+$ sedge slashing-export --help
+
+Export Slashing Protection Interchange Format (EIP-3076) data. This command assumes
+that the validator client container exists, stopped or not and that its database
+is already initialized. Take in mind that the validator client generates slashing
+protection data after some time running, so for some clients export slashing protection
+data just after start the client could produce some errors.
+
+This command stops the validator client during the exporting process due to the
+validator database being locked while it's running but leaves the validator client
+in the same state in which it was found. That means if the validator is running/stopped
+before the export, then the validator will be running/stopped after the command
+is executed, regardless of whether the export fails or not. To force a different
+behavior use --start-validator and --stop-validator flags.
+
+Usage:
+  sedge slashing-export [validator] [flags]
```

Comment:
There is a mismatch between this help and the actual one. Make sure you update this one after updating the Long description as requested

Reply:
Fixed here https://github.com/NethermindEth/sedge/pull/209/commits/3644f27e138267b8ca3c5b615210d9daa2098bf6